#154953782 Add fitbit support

JoyyToo opened the pull request:
#### What does this PR do?
- Add fitbit support 

#### Description of Task to be completed?
- Being able to pull in exercise and weight information from fitbit would be fantastic. The upside of intergrating with Fitbit is that they already allow you to import data from places like calorie counter.

#### How should this be manually tested?
- A user can log in to the fitbit account and create weight entries and activities then access then via the wger application.

#### What are the relevant pivotal tracker stories?
- [154953782](https://www.pivotaltracker.com/story/show/154953782)

#### Screenshots (if appropriate)
<img width="1394" alt="screen shot 2018-03-01 at 09 18 40" src="https://user-images.githubusercontent.com/27849036/36830041-f2f72728-1d32-11e8-8918-0bd5b67eff4f.png">

- Sync weight
<img width="1433" alt="screen shot 2018-03-01 at 09 18 56" src="https://user-images.githubusercontent.com/27849036/36830056-0bba731e-1d33-11e8-9d16-3e4477e0716f.png">

<img width="1421" alt="screen shot 2018-03-01 at 09 25 12" src="https://user-images.githubusercontent.com/27849036/36830082-2b181fd6-1d33-11e8-9a21-f0948da1126c.png">

- Sync exercises
<img width="1391" alt="screen shot 2018-03-01 at 09 24 38" src="https://user-images.githubusercontent.com/27849036/36830100-4008d08e-1d33-11e8-8dce-322885e1c828.png">

<img width="1428" alt="screen shot 2018-03-01 at 09 25 41" src="https://user-images.githubusercontent.com/27849036/36830117-4e26dda0-1d33-11e8-9cc9-f26137dd9c95.png">

#### Questions:

muchai-mercy commented on wger/exercises/urls.py:
```diff
@@ -122,7 +122,7 @@
         name='pending'),
     url(r'^(?P<pk>\d+)/accept/$', exercises.accept, name='accept'),
-    url(r'^(?P<pk>\d+)/decline/$', exercises.decline, name='decline'),
+    url(r'^(?P<pk>\d+)/decline/$', exercises.decline, name='decline'),   
```
rm trailing white space

muchai-mercy commented on wger/core/views/fitbit.py:
```diff
@@ -0,0 +1,116 @@
+    AUTHORIZE_URI = 'https://www.fitbit.com/oauth2/authorize'
+    TOKEN_REQUEST_URI = 'https://api.fitbit.com/oauth2/token'
+
+    def ComposeAuthorizationuri(self):
```
Update to correct naming convention

JoyyToo committed:
[Feature 154953782] Add fitbit support:
  - Add fitbit to requirements
  - Add views for authorizing and getting token from the app
  - Add method to fetch weight data from fitbit
  - Add method to fetch activity data from fitbit
  - Add urls for the views
  - Add buttons for linking with fitbit
